app/src/main/res/layout/fragment_article.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<TextView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Try to scroll long text, which doesn't fit to one screen. You'll find out that the scrolling is sloppy and not smooth. Probably for better performance it is better to wrap `TextView` with `ScrollView`.